packages/cozy-app-publish/README.md

```diff
@@ -60,6 +65,43 @@ yarn cozy-app-publish \
 --manual-version 1.0.2-dev.042cef26d9d33ea604fe4364eaab569980b500c9
 ```
 
+#### Publishing a beta version
+
+Beta versions are only available through beta channel of the registry and are not automatically deployable on production instances. However a beta tester can
+force deployment of beta versions of a given (installed) app by clicking trhee times in a row on the app version in the app store.
```

Comment:
```suggestion
force deployment of beta versions of a given (installed) app by clicking three times in a row on the app version in the Cozy store.
```

Are we sure we want to document publicly that stuff? I've nothing against it. :)

Reply:
Resolved in  4587fd0